src/components/Popups/PopupHighlight.scss

```diff
@@ -0,0 +1,56 @@
+@import '~box-ui-elements/es/styles/variables';
+
+.ba {
+    $popup-arrow-height: 3px;
+    $popup-arror-width: 4px;
+    $popup-content-bg: $bdl-gray;
+    $popup-icon-margin: 8px;
+
+    .ba-PopupHighlight {
+        pointer-events: auto;
+
+        // Top-facing arrows
+        &[data-popper-placement*='bottom'] {
+            .ba-Popup-arrow {
+                top: -$popup-arrow-height;
+            }
+        }
+
+        .ba-Popup-arrow {
```

Comment:
The arrow styling is pretty complex, so we should try to avoid complicating it further, if possible. If we absolutely must change the size, please create a mixin for PopupBase that accepts a size parameter and generates styles as needed rather than duplicating them here.